certbot/display/ops.py

```diff
@@ -289,3 +289,49 @@ def _gen_https_names(domains):
             domains[-1])
 
     return ""
+
+
+def _get_validated(method, validator, message, default=None, **kwargs):
+    if default is not None:
+        try:
+            validator(default)
+        except errors.PluginError as error:
+            raise AssertionError("Invalid default {0}".format(default))
```

Comment:
> This probably will never matter, but you can turn off assertions in Python by passing `-O` to the interpreter, but this doesn't work if you explicitly raise an `AssertionError`. Instead, we could do `assert False, "Invalid default {}".format(default)` here.

I'm not sure being able to turn this off would be good; it seems like that could lead to an infinite loop below.